package.json

```diff
@@ -27,7 +27,7 @@
   "author": "",
   "license": "ISC",
   "scripts": {
-    "test": "jest | coveralls",
+    "test": "jest",
```

Comment:
I think I understand why this change was made. But can you explain here, so we can understand why you are not piping the test results to coveralls in the `test script` anymore

Reply:
@micah-akpan That was an over sight